libs/pages/cluster/src/lib/ui/page-settings-credentials/page-settings-credentials.spec.tsx

```diff
@@ -0,0 +1,90 @@
+import { waitFor } from '@testing-library/react'
+import { render } from '__tests__/utils/setup-jest'
+import { wrapWithReactHookForm } from '__tests__/utils/wrap-with-react-hook-form'
+import { CloudProviderEnum } from 'qovery-typescript-axios'
+import { clusterFactoryMock, credentialsMock, organizationFactoryMock } from '@qovery/shared/factories'
+import { ClusterCredentialsEntity, ClusterEntity, OrganizationEntity } from '@qovery/shared/interfaces'
+import PageSettingsGeneral, { PageSettingsCredentialsProps } from './page-settings-credentials'
+
+const mockCluster: ClusterEntity = clusterFactoryMock(1, CloudProviderEnum.AWS)[0]
+const mockOrganization: OrganizationEntity = organizationFactoryMock(1)[0]
+const mockCredentials: ClusterCredentialsEntity[] = credentialsMock(2)
+
+jest.mock('@qovery/domains/organization', () => {
```

Comment:
I don't get what you have to mock all of that here since we are not in the feature you should not have so much logic stuff on this side no?